File: main/java/com/example/cyclingapp/MainActivity.java
package com.example.cyclingapp;

import androidx.annotation.NonNull;
import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import android.content.Intent;
import android.os.Bundle;
import android.view.Menu;
import android.view.MenuInflater;
import android.view.MenuItem;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ActionBar actionBar = getSupportActionBar();

        if (actionBar != null) {
            actionBar.setTitle("Cycling App");
            actionBar.setDisplayHomeAsUpEnabled(false);
        }
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        MenuInflater inflater = getMenuInflater();
        inflater.inflate(R.menu.three_dot_menu, menu);
        return true;
    }

    @Override
    public boolean onOptionsItemSelected(@NonNull MenuItem item) {

        Intent intent;
        switch(item.getItemId()){

            case R.id.user_dot:
                intent = new Intent(this, UserProfileActivity.class);
                startActivity(intent);
                return true;

            case R.id.settings_dot:
                //Toast.makeText(this, "general settings selected",Toast.LENGTH_SHORT).show();
                intent = new Intent(this, GeneralSettingsActivity.class);
                startActivity(intent);
                return true;

            default:
                return super.onOptionsItemSelected(item);
        }

    }
}

File: src/main/java/com/example/exerciseapp/MainActivity.kt
package com.example.exerciseapp

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() { //you're coding in kotlin, different syntax




    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        openUserInfo()
        openGeneralSettings()



    }

    private fun openGeneralSettings() : Unit {

        val settingsButton:Button = findViewById<Button>(R.id.display_settings_button)
        settingsButton.setOnClickListener{

            val intent: Intent = (Intent(this, GeneralSettingz::class.java))
            startActivity(intent)
        }


    }

    private fun openUserInfo() : Unit {
        val userInfoButton:Button = findViewById<Button>(R.id.user_info_button)
        userInfoButton.setOnClickListener {

            val intent: Intent = (Intent(this, UserInfo::class.java))
            startActivity(intent)
        }


    }
}